torchao/quantization/quant_api.py

```diff
@@ -21,7 +21,11 @@
 import torch.nn.functional as F
 from typing import Any, Callable, Union, Dict, Optional
 
-from torchao.dtypes import PlainLayoutType
+from torchao.dtypes import (
+    to_affine_quantized, 
```

Comment:
thanks, if there is no circular dep you can remove the import from other functions as well, e.g. `int8_weight_only`

Reply:
done